### ISSUE TITLE
Add more rectangular cosine masking

### ISSUE THREAD

**Issue by mjo22** [mjo22/cryojax#280]
Right now, cryojax.operators.masks has some rectangular cosine masking but it is not complete. It has a 2D square mask and a 2D "cylindrical" mask (it is just a rotated rectangular). I already basically have the code to add a 3D rectangular mask and a 2D rectangular mask, so I may as well round out the rectangular masking functionality in 2D. This would also begin on the 3D case, which I would complete at some other date

**Comment by mjo22** [comment]
Closes with #382
